codersdk/deployment.go

```diff
@@ -2611,6 +2613,16 @@ Write out the current server config as YAML to stdout.`,
 		emailTLSCertFile,
 		emailTLSCertKeyFile,
 		// Notifications Options
+		{
+			Name:        "Notifications: Enabled",
+			Description: "Controls if notifications are enabled.",
```

Comment:
```suggestion
			Description: "Enable or disable notifications.",
```
Suggestion: Simplify